docs/cookbook-recipes/isle-v140-git-cleanup.md

```diff
@@ -0,0 +1,163 @@
+* Work from the newly resized and smaller forked or cloned ISLE project directory.
+  
+### Git cleanup process - Steps
+
+* Open up a terminal
+
+* Change directory to a location of your choice
+  * `cd your_path_here`
+
+* Make a new temporary direcotory called `ISLE_140_GIT_CLEANUP`
+    * `mkdir ISLE_140_GIT_CLEANUP`
+
+* Download the script found and save it to the `ISLE_140_GIT_CLEANUP` directory and fix the permissions.
+  * Right click on this link below and click "Save link as"
+    * [isle-v140-git-cleanup-script.sh](../assets/isle-v140-git-cleanup-script.sh)
+  * `chmod 755 isle-v140-git-cleanup-script.sh`
+
+* Run this git command to clone your forked or cloned ISLE project directory. This is not the usual `git clone` command, note the `--mirror` flag.
+  * `git clone --mirror git project url`
+  * Example:
+    * `git clone --mirror git@github.com:username/ISLE.git` (_as a fork using the same project name or title_)
+    * `git clone --mirror git@github.com:username/acmeorg_isle.git` (_as a fork or cloned repo using a different project name or title_)
+  * This will create a new directory called `ISLE.git` or `acmeorg_isle.git` which is not typical clone or fork output.
+
+* Run the cleanup script
+  * `./isle-v140-git-cleanup-script.sh`
+
+* At the start of the script, there will be an interactive input prompt asking you to:
+  
+```bash
+"Type in the name of your git repo only e.g. ISLE (do not include .git) and then press the [ENTER] or [RETURN] key to continue"`
+Enter the name here:  
+```
+* Enter the name of your forked or cloned ISLE git repository without the .git extension e.g `ISLE` or `acmeorg-isle` etc.
+
+* The script will continue and the process takes about 30 - 45 seconds depending on the speed of your system with minimal output.
+
+* Your final step is to push this local smaller repo back to the remote repository
+* `git push origin master`
+
+* Clone down your newly changed and smaller forked or cloned ISLE project directory again to ensure that the repositiory is smaller to a location of your choice.
+
+- Remove all previous larger size versions of your forked or cloned ISLE project directory. You can now delete the `ISLE_140_GIT_CLEANUP` directory and its contents.
+
+* Work from the newly resized and smaller forked or cloned ISLE project directory.
+
+---
+
+
+## Identification tools
+
+If you would like to check if the files have been removed or are still present:
+
+* Clone this [repo](https://github.com/ivantikal/git-tools.git) to pinpoint offending files and folders to the parent directory of where your local forked or cloned ISLE git repository is located. **Do not** clone this tool into your local forked or cloned ISLE git repository.
+  * `git clone https://github.com/ivantikal/git-tools.git`
+  * Your parent directory should now look like this: (_again please note your local forked or cloned ISLE git repository may not use the ISLE as the title_)
```

Comment:
may not use ISLE as the title

Reply:
^^ aside from Mark's edit, I don't really get what you're saying here.  It says it may not use ISLE as the title, but the example uses ISLE, so there's something I'm not understanding here.

Reply:
I guess I'm clumsily trying to anticipate that someone's project may not be named ISLE and some who will have it named ISLE. I'll amend the comment.

Reply:
`may not use` should really be `might not use`. It isn't a permission thing just bad wording on my part.